packages/front-end/tests/e2e/lab-manager/Labs_Lab_Manager.spec.e2e.ts

```diff
@@ -0,0 +1,38 @@
+import { test, expect } from 'playwright/test';
+
+test('01 - Create a Laboratory Unsuccessfully', async ({ page, baseURL }) => {
```

Comment:
Should this read "Successfully" instead of "Unsuccessfully"?

Reply:
This one is to check that user is not able to create a Lab, it only checks for the 'Create a new lab' button

Reply:
Ok, in this case I think it makes more sense as simply "Laboratory creation button is hidden".

Otherwise it reads as if the user has tried to create a Lab unsuccessfully.

Reply:
Script titles were updated